index.js

```diff
@@ -213,7 +213,7 @@ async function fastifyStatic (fastify, opts) {
         if (opts.list && dirList.handle(pathname, opts.list)) {
           dirList.send({
             reply,
-            dir: dirList.path(opts.root, pathname),
+            dir: Array.isArray(opts.root) ? opts.root[0] : dirList.path(opts.root, pathname),
```

Comment:
This statement does not apply to the [least astonishment principale](https://en.wikipedia.org/wiki/Principle_of_least_astonishment).

The user would expect that the array should be processed, but it is not

I think we have two options:

1. manage the array
2. throw an error as multi-root is not supported

Reply:
@Eomm 

Thanks.

I guess option No. 2 would be nice because the directory listing feature would be confusing if it supports multiple `root`. So, I'd like to add validation to the `validateOptions` function. What do you think about it?

Reply:
@Eomm 

I updated it. Could you review it again?